### DISCUSSION TITLE
How to create a database?

### DISCUSSION THREAD

**Discussion by YunshuaiPeng** [planetscale/discussion#20] [Q&A] [answered]
```bash
xxx/development> CREATE DATABASE test;
ERROR 1235 (42000): create database is not supported
```
I read the official document carefully, but I can't find any useful infomation.

**Comment by bigint** [comment]
@pys1992 The database is already created, in this case `development` is your database name 😄

**Reply by YunshuaiPeng** [reply]
Actually all I can see is onboarding-demo database except built-in's.
```bash
xxx/development> SHOW DATABASES;
+--------------------+
| Database           |
+--------------------+
| onboarding-demo    |
| information_schema |
| mysql              |
| sys                |
| performance_schema |
+--------------------+
```
![image](https://user-images.githubusercontent.com/44765369/118784668-ca2c5000-b8c2-11eb-8191-66a709002146.png)

**Comment by Anders-PlanetScale** [comment, marked as answer]
@pys1992 when you create a DB with the onboarding-demo box checked the onboarding-demo DB replaces the empty DB that would normally get created.

You will need to create a new DB without the onboarding-demo box checked, you can also use `pscale database create <database name>`

**Reply by YunshuaiPeng** [reply]
It works. Thanks a lot.

**Comment by muhaimincs** [comment]
Hi, when I tried this

```bash
~ % pscale database create my-db 
Database my-db was successfully created.
```

but when I checked

```bash
xxx/development> SHOW DATABASES;
+--------------------+
| Database           |
+--------------------+
| onboarding-demo    |
| information_schema |
| mysql              |
| sys                |
| performance_schema |
+--------------------+
```

please help

**Reply by Anders-PlanetScale** [reply]
@muhaimincs when you create a DB with the onboarding-demo box checked the onboarding-demo DB replaces the empty DB that would normally get created.

You will need to create a new DB without the onboarding-demo box checked, you can also use pscale database create <database name>

**Reply by muhaimincs** [reply]
Thanks. That's help
